config/jobs/kubernetes/kops/build-pipeline.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+
+template = """
+# Verify the latest-ci version from the {{branch}} branch of kops
+# Runs a small subset of the e2e tests.
+# Publishes the version to latest-ci-updown-green on success.
+- interval: 60m
+  name: {{name}}
+  decorate: true
+  decoration_config:
+    timeout: 45m
+  labels:
+    preset-service-account: "true"
+    preset-aws-ssh: "true"
+    preset-aws-credential: "true"
+  spec:
+    containers:
+    - image: {{e2e_image}}
+      command:
+      - runner.sh
+      - kubetest
+      args:
+      # Generic e2e test args
+      - --up
+      - --test
+      - --down
+      - --dump=$(ARTIFACTS)
+      - --timeout=45m
+      - --gcp-service-account=$(E2E_GOOGLE_APPLICATION_CREDENTIALS)
+      # kops-specific test args
+      - --deployment=kops
+      - --provider=aws
+      - --cluster={{name}}.test-cncf-aws.k8s.io
+      - --kops-ssh-user={{ssh_user}}
+      - --kops-nodes=4
+      - --extract={{extract}}
+      - --kops-state=s3://k8s-kops-prow/
+      - --kops-ssh-key=$(AWS_SSH_PRIVATE_KEY_FILE)
+      - --kops-ssh-public-key=$(AWS_SSH_PUBLIC_KEY_FILE)
+      - --kops-publish=gs://k8s-staging-kops/kops/releases/markers/{{branch}}/latest-ci-updown-green.txt
+      - --kops-version=https://storage.googleapis.com/k8s-staging-kops/kops/releases/markers/{{branch}}/latest-ci.txt
+      #- --kops-kubernetes-version should be inferred by kubetest from --extract
+      #- --kops-zone should be randomized by kubetest
+      # Specific test args
+      - --test_args=--ginkgo.focus=\\[k8s.io\\]\\sNetworking.*\\[Conformance\\]
```

Comment:
Maybe skip the slow tests.
```suggestion
      - --test_args=--ginkgo.focus=\\[k8s.io\\]\\sNetworking.*\\[Conformance\\] --ginkgo.skip=\\[Slow\\]|\\[Serial\\]
```